pandas/core/generic.py

```diff
@@ -4951,10 +4951,12 @@ def pipe(self, func, *args, **kwargs):
     Returns
     -------
     DataFrame, Series or scalar
-        If DataFrame.agg is called with a single function, returns a Series
-        If DataFrame.agg is called with several functions, returns a DataFrame
-        If Series.agg is called with single function, returns a scalar
-        If Series.agg is called with several functions, returns a Series
+        The return can be:
+        - If DataFrame.agg is called with a single function, returns a Series.
```

Comment:
does something check if we have these as bullets (or not)?

Reply:
Just fix them. It wan't also rendering very well.